mmdeploy/codebase/mmedit/deploy/super_resolution_model.py

```diff
@@ -104,6 +110,17 @@ def forward_test(self,
         """
         outputs = self.forward_dummy(lq)
         result = self.test_post_process(outputs, lq, gt)
+
+        # Align to mmediting BasicRestorer
+        if save_image and save_path:
```

Comment:
seems `save_image` and `save_path` both need to be specified if I want to save. Keeping only `save_path` is enough then.

Reply:
I agree. Our `tools/test.py` uses `--show` and `--show-dir` to control the output behavior. Maybe it should also be simplified later.